scripts/bootstrap.ps1

```diff
@@ -37,12 +39,30 @@ if ((get-module psake) -eq $null) {
 }
 
 if ((get-module psake) -eq $null) {
-    # Still not loaded, let's try the various NuGet caches
-    $locals = .\nuget locals all -list
+    # Not yet loaded, try to load it from the chocolatey installation library
+    TryLoad-Psake $env:ProgramData\chocolatey\lib
+}
+
+if ((get-module psake) -eq $null) {
+    # Still not loaded, let's download it and then look in the various NuGet caches
+
+    pushd .\src\sestest
+    dotnet add package psake | Out-Null
```

Comment:
does `dotnet add` not alter the projects?

Reply:
Technically yes, but `psake` is already referenced so there's no actual change. This was the cleanest way I could see to provoke `dotnet` into downloading the package.